src/utils/general.js

```diff
@@ -67,7 +67,7 @@ const reverseString = str => str.split('').reverse().join('')
 
 const isLastDeclarationCompleted = text => {
   // We disregard all comments in this assessment of declaration completion
-  const commentsRemoved = text.replace(/\/\*.*?\*\//g, '')
+  const commentsRemoved = text.replace(/\/\*[\s\S]*?(?!\*\/)?\*\//g, '')
```

Comment:
What is this supposed to do `(?!\*\/)?`? Is it supposed to be a conditional negative lookahead? Doesn't that just match everything then? I also don't think you need it, the `*?` is a non-greedy matcher so it'll stop as soon as it can

Reply:
I played around a bit more in regexr and managed to shorten it a little to `/\/\*(?!\*\/)[\s\S]*?\*\//g`. This is doing a negative lookahead of the sequence `*/` and then lazily matches any other characters. Without the `?` it would match everything between multiple comment blocks.